modules/dxp/apps/segments/segments-experiment-web/src/main/resources/META-INF/resources/js/components/ReviewExperimentModal.es.js

```diff
@@ -148,32 +151,95 @@ function ReviewExperimentModal({modalObserver, onModalClose, onRun, variants}) {
 				) : (
 					<div ref={measureHeight}>
 						<h3 className="border-bottom-0 sheet-subtitle text-secondary">
+							{Liferay.Language.get('test-type')}
+						</h3>
+
+						<hr />
+
+						<p className="small">
+							{Liferay.Language.get(
+								'choose-how-your-experiment-is-going-to-be-measured'
+							)}
+						</p>
+
+						<label>{Liferay.Language.get('standard')}</label>
+
+						<p className="small">
+							{Liferay.Language.get(
+								'use-the-ab-test-methodology-to-determine-which-variant-performs-better-based-on-statistical-significance,-it-requires-high-traffic-to-reach-conclusive-results'
+							)}
+						</p>
+
+						<ClayRadioGroup
+							inline
+							onChange={(value) => {
+								setSelectedTestType(value);
+							}}
+							value={selectedTestType}
+						>
+							<ClayRadio
+								label={Liferay.Language.get(
+									'enable-standard-test-type'
+								)}
+								value="AB"
+							></ClayRadio>
+						</ClayRadioGroup>
+
+						<h3 className="border-bottom-0 mt-3 sheet-subtitle text-secondary">
 							{Liferay.Language.get('traffic-split')}
 						</h3>
 
 						<SplitPicker
 							onChange={(variants) => {
 								setDraftVariants(variants);
 							}}
+							selectedTestType={selectedTestType}
 							variants={draftVariants}
 						/>
 
-						<hr />
-
 						<h3 className="border-bottom-0 sheet-subtitle text-secondary">
 							{Liferay.Language.get('confidence-level')}
 						</h3>
 
 						<SliderWithLabel
+							disabled={selectedTestType === 'MAB'}
 							label={Liferay.Language.get(
 								'confidence-level-required'
 							)}
 							max={MAX_CONFIDENCE_LEVEL}
 							min={MIN_CONFIDENCE_LEVEL}
 							onValueChange={setConfidenceLevel}
+							selectedTestType={selectedTestType}
```

Comment:
You don't need to pass selectedTestType here because you already have disabled prop

Reply:
done